src/seedu/addressbook/data/person/Address.java

```diff
@@ -55,4 +55,9 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+
+    @Override
+    public String getPrintableString() {
+        return "Address: " + toString();
```

Comment:
toString() can be replaced by value for readability. toString() will be called implicitly.